Neos.Neos/Classes/FrontendRouting/DimensionResolution/DelegatingResolver.php

```diff
@@ -52,36 +52,32 @@ public function fromRequestToDimensionSpacePoint(
             );
         }
         $siteConfiguration = $site->getConfiguration();
-        $factory = $this->objectManager->get(
-            $siteConfiguration['contentDimensions']['resolver']['factoryClassName']
-                ?? throw new \RuntimeException(
-                    'No Dimension Resolver Factory configured at'
-                        . ' Neos.Neos.sites.*.contentDimensions.resolver.factoryClassName'
-                )
-        );
+
+        $factory = $this->objectManager->get($siteConfiguration->contentDimensionResolverFactoryClassName);
         assert($factory instanceof DimensionResolverFactoryInterface);
-        $resolverOptions = $siteConfiguration['contentDimensions']['resolver']['options'] ?? [];
         $context = $factory->create(
             $siteDetectionResult->contentRepositoryId,
-            $resolverOptions
+            $siteConfiguration
         )->fromRequestToDimensionSpacePoint($context);
 
-        return self::fillWithDefaultDimensionSpacePoint(
+        return $context;
+        // TODO CHANGE ME
+        /*return self::fillWithDefaultDimensionSpacePoint(
             $context,
-            $siteConfiguration['contentDimensions']['defaultDimensionSpacePoint'] ?? []
-        );
+            $siteConfiguration->defaultDimensionSpacePoint
+        );*/
```

Comment:
I have removed this, because we need a way to express "X cannot match..." -> need to discuss this. maybe extend the matching logic.